Documentation/operations/performance/scalability/identity-relevant-labels.rst

```diff
@@ -21,19 +21,21 @@ application.
 By default, Cilium considers all labels to be relevant for identities, with the
 following exceptions:
 
-================================== ==============================================
-Label                               Description
----------------------------------- ----------------------------------------------
-``any:!io.kubernetes``             Ignore all ``io.kubernetes`` labels
-``any:!kubernetes\.io``            Ignore all other ``kubernetes\.io`` labels
-``any:!beta.kubernetes\.io``       Ignore all ``beta\.kubernetes\.io`` labels
-``any:!k8s\.io``                   Ignore all ``k8s\.io`` labels
-``any:!pod-template-generation``   Ignore all ``pod-template-generation`` labels
-``any:!pod-template-hash``         Ignore all ``pod-template-hash`` labels
-``any:!controller-revision-hash``  Ignore all ``controller-revision-hash`` labels
-``any:!annotation.*``              Ignore all ``annotation`` labels
-``any:!etcd_node``                 Ignore all ``etcd_node`` labels
-================================== ==============================================
+============================================= =====================================================
+Label                                         Description
+--------------------------------------------- -----------------------------------------------------
+``any:!io.kubernetes``                        Ignore all ``io.kubernetes`` labels
+``any:!kubernetes\.io``                       Ignore all other ``kubernetes\.io`` labels
+``any:!statefulset\.kubernetes\.io/pod-name`` Ignore ``statefulset\.kubernetes\.io/pod-name`` label
```

Comment:
```suggestion
``any:!statefulset\.kubernetes\.io/pod-name`` Ignore ``statefulset.kubernetes.io/pod-name`` label
```
Formatting is already inconsistent in this table, so I wouldn't block on this suggestion really.